pkg/webhooks/trainingruntime_webhook.go

```diff
@@ -54,15 +56,26 @@ func (w *TrainingRuntimeWebhook) ValidateCreate(ctx context.Context, obj apirunt
 }
 
 func validateReplicatedJobs(rJobs []jobsetv1alpha2.ReplicatedJob) field.ErrorList {
+	ancestors := sets.New(constants.AncestorTrainer, constants.ModelInitializer, constants.DatasetInitializer)
 	rJobsPath := field.NewPath("spec").
 		Child("template").
 		Child("spec").
 		Child("replicatedJobs")
 	var allErrs field.ErrorList
 	for idx, rJob := range rJobs {
-		if rJob.Replicas != 1 {
+		if rJob.Name == constants.Launcher && rJob.Replicas != 1 {
 			allErrs = append(allErrs, field.Invalid(rJobsPath.Index(idx).Child("replicas"), rJob.Replicas, "always must be 1"))
+			continue
```

Comment:
```suggestion
```

We want to verify all validation errors by one request.

Reply:
Thanks, I'll fix it.